### DISCUSSION TITLE
Able to block a contact on mobile app

### DISCUSSION THREAD

**Discussion by nguyenkims** [simple-login/app#669] [Feature Requests]
Recently on the web app, we can block a sender.

![](https://i.redd.it/hxzecltun5w71.png)

It'd be great to have this feature on mobile too :).

**Comment by nguyenkims** [comment]
@ntnhon for info :)

**Reply by ntnhon** [reply]
Great! Did you also update the GET /api/aliases/:alias_id/contacts?

**Reply by nguyenkims** [reply]
@ntnhon Yeah this endpoint is now returning a new field `block_forward` that indicates whether a contact is blocked.
https://github.com/simple-login/app/blob/master/docs/api.md#get-apialiasesalias_idcontacts

To block/unblock a contact, there's this new endpoint https://github.com/simple-login/app/blob/master/docs/api.md#post-apicontactscontact_idtoggle which is similar to the alias toggle one.

**Comment by EdgeCon** [comment]
Has the feature to block individual senders, from the android/iOS app, been suggested yet?

**Reply by ntnhon** [reply]
Yes you can now block individual contacts in both iOS and Android apps. For now you can only do it in the contact list page. Not in the alias detail page.

**Reply by EdgeCon** [reply]
Thanks! I have been looking for that feature for a while now. Dang, they got to put that useful feature in the alias details page too.

**Reply by sloth5649** [reply]
Sorry for the stupid question. However I can't really figure out how to access the contact list page in the Android app (v. 1.19.2)

**Reply by ntnhon** [reply]
@sloth5649 Hey, please understand that we're talking about contact list of aliases, not the contact list of your phone. To view the contact list of aliases, tap on "Send email" from the homepage. From there tapping on any contact will reveal 4 options, "Block" is one of them.

**Reply by Mincka** [reply]
Was it removed at some point? I can only see 3 options for a sender: copy, copy without name, compose with default client.

**Reply by Yrlish** [reply]
These is no option to block individual senders in an alias (on Android 1.20.1). The previously mentioned menu only has three options, not a fourth for "block".

@ntnhon do you have anything?
